### DISCUSSION TITLE
Third-party Syncplay forks/implementations

### DISCUSSION THREAD

**Discussion by Et0h** [Syncplay/syncplay#574] [Show and tell]
_**Note: If you have an issue with a third-party implementation of Syncplay then please use their Issues page rather than raising the issue with the Syncplay team!**_

### Active open source third-party implementations of Syncplay (i.e. latest updated in 2022 or later):

- **Syncplay Mobile** by @yuroyami which can be found at https://github.com/yuroyami/syncplay-mobile/
- **Kodi** by @Skaytacium at https://github.com/Skaytacium/synko

### Inactive open source third-party implementations of Syncplay (i.e. last updated in 2021 or earlier):

- **Syncplay.net** by @arkangel-dev at https://github.com/arkangel-dev/SyncPlay.NET (September 2021)
- **Syncplayer for Android** by @mo3rfan at https://github.com/mo3rfan/syncplayer (February 2021)
- **Syncplay Web** by @mo3rfan at https://github.com/mo3rfan/syncplay-web (April 2017)
- **Syncplay Java** by @mo3rfan at https://github.com/mo3rfan/syncplay-java (November 2017)

### Potentially helpful forks/tweaks/builds:
- Syncplay proxies and reverse proxies: https://github.com/Syncplay/syncplay/issues/346
- Unofficial docker images maintained by @oulh: https://github.com/Syncplay/syncplay/discussions/523 and https://hub.docker.com/r/oulh/syncplay-server/tags
- Unofficial openSUSE package maintained by @nopeinomicon: https://software.opensuse.org/package/syncplay
- Unofficial Fedora rpm packages maintained by @batmanfeynman: https://copr.fedorainfracloud.org/coprs/batmanfeynman/syncplay/

### Details on the Sycplay protocol:
- https://syncplay.pl/about/protocol/
- https://github.com/Reddnek/syncplay-android/issues/6
- https://github.com/Reddnek/syncplay-android/issues/17
- https://github.com/Syncplay/syncplay/discussions/542

### Guides for developing the main Syncplay client/server:
- [Adding support for a new media player to Syncplay](https://github.com/Syncplay/syncplay/wiki/Guide-to-adding-support-for-a-new-media-player-to-Syncplay)
- [Guide to translating Syncplay into another language](https://github.com/Syncplay/syncplay/wiki/Guide-to-translating-Syncplay-into-another-language)

### Other useful guides
- See https://syncplay.pl/guide/server/ for guides on running Syncplay servers under various circumstances.

**Comment by yuroyami** [comment]
Thank you for the mention :) I would love to address a little change here

syncplay-android is now [**syncplay-mobile**](https://github.com/yuroyami/syncplay-mobile/). This brings the same functionality the same app on Android to iOS.

Edit: Even those with nonjailbroken iOS devices can install the app via AltStore as of 2026.

**Comment by everpcpc** [comment]
https://github.com/everpcpc/syncplay-tauri

Another client written with Tauri.
